### ISSUE TITLE
MySQL Isolation levels doesn't work( DBeaver v3.6.7)

### ISSUE THREAD

**Issue by RichieHsu** [serge-rider/dbeaver#677]
MySQL version： 5.0.51a-24+lenny4
OS：Debian
JDBC Driver：mysql-connector-java-5.1.39-bin.jar
DBeaver version：v3.6.7

Problem Describe：
    I set "Isolation level " to Read committed and tick off the "Auto-commit".
But It still auto commit the insert, update statment which I issued.
I had tried Repeatable reads, Read uncommitted, none of them works.

Don't know what's going wrong?

![dbeaver1](https://cloud.githubusercontent.com/assets/12066066/17920402/e2f42c80-6a06-11e6-9166-03d95967052f.png)

![dbeaver2](https://cloud.githubusercontent.com/assets/12066066/17920408/eb63797a-6a06-11e6-915d-4778443c58bd.png)

**Comment by serge-rider** [comment]
Can't reproduce.
How do you know that it is in auto-commit, how do you test it?

**Comment by RichieHsu** [comment]
Here is the way I test：
(1)DBeaver database option is not setted with auto-commit：
![1](https://cloud.githubusercontent.com/assets/12066066/18038248/1cec80f8-6dc8-11e6-9473-81b98b98ed12.png)

(2)select @@autocommit;
the atuocommit mode is 0   ( which means autocommit is off ):
![2](https://cloud.githubusercontent.com/assets/12066066/18038250/240a502c-6dc8-11e6-88ba-84e773d68e27.png)

(3)Create a table "richie_test),  insert a row and rollback after inserted the row:
create table richie_test(
  col  varchar(10)
);

insert into richie_test(col)
values
( 'abc');

rollback;

(4)Close the connection and re-connect. select the table again.
select * from richie_test;

the row which I inserted is not rollbacked.

**Comment by serge-rider** [comment]
Still can't reproduce.
BTW why do you use obsolete MySQL driver (3.x and earlier)? You can use just standard driver.
Did you try to use DBeaver Commit/Rollback actions (instead of commit/rollback SQL commands)?

**Comment by RichieHsu** [comment]
I had tried to use DBeaver Commit/Rollback actions.  still not works.
Maybe the problem is on DBMS.  It has same problem when I tried to use MySQL Workbench 6.3 CE.
Will try others DB Client Tools latter.

BTW, Is this the standard driver you mentioned?
I use MySQL 3.x deiver is because this one is not work. There are only two driver I can choose.
That's why I use it... 

Any suggestion? Thanks.

![1](https://cloud.githubusercontent.com/assets/12066066/18113241/9bc2da46-6f60-11e6-8ce3-d024ef5eb1b1.png)

![2](https://cloud.githubusercontent.com/assets/12066066/18113386/fa5e37c0-6f61-11e6-8944-c63ac24c9334.png)

![3](https://cloud.githubusercontent.com/assets/12066066/18113390/022b8e76-6f62-11e6-88fa-5232651ecbad.png)
